### DISCUSSION TITLE
Github copilot wont work even if I have the latest version of vs code

### DISCUSSION THREAD

**Discussion by SaptakBhoumik** [community/community#6880] [Copilot Conversations] [answered]
I installed copilot today but I am getting the following error 

```
GitHub Copilot requires VS Code 1.57+. Please update your VS Code
```

But I already have the latest version of vs code

**Comment by davecheney** [comment, marked as answer]
@SaptakBhoumik can you please double check that you're using VSCode, not VScodium. This could be tricky if you're using a version bundled with a linux distribution. For example I think Arch installs VSCodium by default.

**Reply by SaptakBhoumik** [reply]
@davecheney I am using vs code not vs codium

**Reply by davecheney** [reply]
@SaptakBhoumik thanks for your reply. Would you be able to post a screenshot -- or just the text, of the About Visual Studio Code dialog box. Thank you.

**Reply by SaptakBhoumik** [reply]
![image](https://user-images.githubusercontent.com/74509281/139193367-a0b5ba8b-069c-4030-98f8-1eb56ca2649c.png)

**Reply by davecheney** [reply]
@SaptakBhoumik thank you. I'm not an expert but as far as I can tell Code OSS is not VSCode. I suggest following these instructions to install vscode for linux. https://code.visualstudio.com/docs/setup/linux

**Reply by SaptakBhoumik** [reply]
Oo ok thanks @davecheney

**Reply by davecheney** [reply]
Hope that helped; the vscode on linux story does seem to be a bit confusing.

**Reply by SaptakBhoumik** [reply]
@davecheney Yes it is working now. Thank you very much

**Reply by ghost** [reply]
visual-studio-code-bin install this

**Reply by ragnarok22** [reply]
> visual-studio-code-bin install this

I will try this and tell
